docs/content/specs/bicep/_index.md

```diff
@@ -583,3 +583,95 @@ To leverage the feature you must
 ---
 
 <br>
+
+#### ID: BCPNFR17 - Category: Composition - Code Styling - Type casting
+
+To improve the usability of primitive module properties declared as strings, you should declare them as the type they are and apply any required casting in the module on behalf of the user.
```

Comment:
Although I know what the goal is, I'm not sure if "as the type they are" clarifies. I don't have a good suggestion either, but trying: 
```suggestion
To improve the usability of primitive module properties declared as strings, you should declare them using a type which better represents them, and apply any required casting in the module on behalf of the user.
```